specification/security/resource-manager/Microsoft.Security/preview/2020-08-06-preview/iotSensors.json

```diff
@@ -231,6 +231,58 @@
           }
         }
       }
+    },
+    "/{scope}/providers/Microsoft.Security/iotSensors/{iotSensorName}/downloadResetPassword": {
+      "post": {
+        "x-ms-examples": {
+          "Download file for reset password of the sensor": {
+            "$ref": "./examples/IotSensors/DownloadResetPassword.json"
+          }
+        },
+        "tags": [
+          "IoT Sensors"
+        ],
+        "operationId": "IotSensors_DownloadResetPassword",
+        "description": "Download file for reset password of the sensor",
+        "produces": [
+          "application/zip"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/IotSensorScope"
+          },
+          {
+            "$ref": "#/parameters/IotSensorName"
+          },
+          {
+            "in": "body",
+            "name": "applianceId",
+            "description": "The appliance id of the sensor.",
+            "required": true,
+            "schema": {
+              "type": "string"
```

Comment:
The request body should be JSON (i.e. { "applianceId": "value" })

Reply:
How should I define it in order to make it a Json?